drivers/nutdrv_qx_hunnox.h

```diff
@@ -0,0 +1,29 @@
+/* nutdrv_qx_zinto.h - Subdriver for Zinto protocol based UPSes
```

Comment:
@marianojan : if this was adapted from a subdriver for another UPS model, worth mentioning taht but the title here should point to Hunnox ;)